clients/java/src/main/java/io/camunda/zeebe/client/impl/response/UpdateTimeoutJobResponseImpl.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.zeebe.client.impl.response;
+
+import io.camunda.zeebe.client.api.response.UpdateTimeoutJobResponse;
+import io.camunda.zeebe.gateway.protocol.GatewayOuterClass;
+
+public class UpdateTimeoutJobResponseImpl implements UpdateTimeoutJobResponse {
+
+  public UpdateTimeoutJobResponseImpl(final GatewayOuterClass.UpdateJobTimeoutResponse response) {}
```

Comment:
❓ We should provide a response to the client ?

Reply:
If I'm not mistaken, the [response is always empty](https://github.com/camunda/zeebe/blob/main/gateway-protocol/src/main/proto/gateway.proto#L556-L557)? Would there be anything we need to add here in that case?

Reply:
Yes, I saw that. My question is more general 😄 We should provide some infos in the response to the clients ?

Reply:
💭 It might be useful for the user to know the new deadline, but if we want to add this it can also be done in a later iteration.

Reply:
I tend to agree with @korthout here. It's something we could add later if we see a need or get such feedback.

Reply:
As per Remco's advise the response is currently empty (the same as with update retries) but if it needs to be changed I can do so.